Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -1595,36 +1723,36 @@ private void OnDestroySessionCompleteCallback(ref DestroySessionCallbackInfo dat
-            // TODO: Validate that this gets to the members/owners of the session in time, and that we haven't already deleted the local information needed to get session information
+            // Before removing the Session from our local data, we need to inform the Owner of the Session that we've left the Session, if we're not the Owner
+            // TODO: Validate that this gets to the members/owners of the Session in time, and that we haven't already deleted the local information needed to get Session information
             string sessionName = (string)data.ClientData;
             Session localSession;
 
             if (!TryGetSession(sessionName, out localSession) || localSession.ActiveSession == null)
             {
-                Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(OnDestroySessionCompleteCallback)}): Could not find local Session and associated ActiveSession, so could not inform owner/members of destruction.");
+                Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(OnDestroySessionCompleteCallback)}): Could not find local Session and associated ActiveSession, so could not inform Owner/members of destruction.");
                 return;
             }
 
             ActiveSessionCopyInfoOptions copyOptions = new ActiveSessionCopyInfoOptions() { };
             Result localCopyResult = localSession.ActiveSession.CopyInfo(ref copyOptions, out ActiveSessionInfo? outActiveSessionInfo);
 
-            // If we were unable to copy the active session's information, or it failed to populate the SessionDetails inside the ActiveSessionInfo, then we can't get the Owner
-            // If we can't get the Owner, we can't determine who should be messaged, or if we are the owner of this Session
+            // If we were unable to copy the active Session's information, or it failed to populate the SessionDetails inside the ActiveSessionInfo, then we can't get the Owner
+            // If we can't get the Owner, we can't determine who should be messaged, or if we are the Owner of this Session
             if (localCopyResult != Result.Success || !outActiveSessionInfo.HasValue || !outActiveSessionInfo.Value.SessionDetails.HasValue)
             {
-                Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(OnDestroySessionCompleteCallback)}): Failed to copy local information for session {sessionName}, so could not inform owner/members of destruction. Result code {localCopyResult}");
+                Debug.LogError($"{nameof(EOSSessionsManager)} ({nameof(OnDestroySessionCompleteCallback)}): Failed to copy local information for Session {sessionName}, so could not inform Owner/members of destruction. Result code {localCopyResult}");
                 return;
             }
 
-            if (outActiveSessionInfo.Value.SessionDetails.Value.OwnerUserId.Equals(EOSManager.Instance.GetProductUserId()))
+            if (localSession.IsLocalUserOwnerOfSession)
```

Comment:
I can see why you wrote this property. I still think that it should either be cached or converted to a static function as I commented earlier, but it's clear why implementing it one way or another is a win. Nice job!